jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/CaseRuntimeDataServiceImpl.java

```diff
@@ -740,6 +741,27 @@ private void getMilestones(NodeContainer container, Collection<CaseMilestone> re
         return result;
     }
     
+    private Map<String, List<String>> collectDataAccessRestrictions(Process process) {
+        // expected format: item:role1,role2;another:role2,role3
+        // where item and item2 are the case file data item names and role1, role2, role3 are case roles
```

Comment:
I cannot see item2 here, just another, so I would update the comment so it is not so confusing.

Reply:
fixed